porting/UBports_installer.rst

```diff
@@ -3,4 +3,6 @@ Configuring the UBports installer
 
 *Work in progress*
 
-For the time being, refer to the UBports porting notes on `the UBPorts installer and UBports recovery <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image>`_.
\ No newline at end of file
+For the time being, Halium 7.1 porters should refer to the UBports porting notes on `the UBPorts installer and System image <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image-(Halium-7.1)>`_.
+
+There is currently no guide that covers these steps for Halium 9 porting.
```

Comment:
I'm not too convinced of the  "currently no guide" statement. By definition the docs are not the authoritative source for what is and isn't documented in the pads and wikis. Assume that tomorrow someone writes a guide for this in the wiki. Now the docs are actually hindering the porting process by telling porters that there is nothing, even though there is.

Reply:
OK. Removed that line.